knitout-to-kcode.js

```diff
@@ -141,9 +141,10 @@ const TYPE_XFER_TWO_PASS = { front:'Xf', back:'Xf' }; //will actually get split
 
 const TYPE_SPLIT_TO_BACK = { kcode: 'Tr-Rr' }; //* //only with type knit_x (not x_knit //?)
 const TYPE_SPLIT_TO_FRONT = {kcode:'Rr-Tr'}; //*
+const TYPE_KNITFRONT_SPLIT2BACK = { kcode: 'Tr-Rr' }; //* //only with type knit_x (not x_knit //?)
+const TYPE_KNITBACK_SPLIT2FRONT = {kcode:'Rr-Tr'}; //*
```

Comment:
`SPLIT2BACK` is inconsistent naming with `SPLIT_TO_BACK`; also unclear if other tokens needed here.

Reply:
yeah, just wanted to create some distinction between the word SPLIT & BACK/FRONT, but TYPE_KNITFRONT_SPLITBACK or TYPE_KNIT_FRONT_SPLIT_BACK would be fine (and more consistent).

other tokens definitely aren't *needed* there; more for keeping track of logic (but can definitely just remove it if you'd prefer not to have the extra code)